### PR TITLE
[CMAKE] target_compile_definitions() calls: Remove (now) useless '-D'

### PR DESCRIPTION
**Pull request by SergeGautherie**
Documented as automatically removed, since CMake 3.12.4 at least.

https://cmake.org/cmake/help/v3.12/command/target_compile_definitions.html

### CODE REVIEW

**Review by encodedpr — APPROVED** [review]

File: sdk/lib/rossym/CMakeLists.txt [hunk 1]
```diff
@@ -13,4 +13,4 @@ list(APPEND SOURCE
     zwfile.c)
 add_library(rossym ${SOURCE})
 add_dependencies(rossym psdk bugcodes)
-target_compile_definitions(rossym INTERFACE "-D__ROS_ROSSYM__")
+target_compile_definitions(rossym INTERFACE "__ROS_ROSSYM__")
```
Comment (HBelusca) [inline sdk/lib/rossym/CMakeLists.txt:16]: Maybe the quotes are not needed too.

**Review by HBelusca — APPROVED** [review]

**Review by zefklop — CHANGES_REQUESTED** [review]
Please remove the extra quotes and this is good to go.

File: sdk/lib/rossym/CMakeLists.txt (hunk 1 above)
Comment (HBelusca) [inline sdk/lib/rossym/CMakeLists.txt:16]:
```suggestion
target_compile_definitions(rossym INTERFACE __ROS_ROSSYM__)
```

Comment (SergeGautherie) [reply sdk/lib/rossym/CMakeLists.txt]: Done.

Comment (SergeGautherie) [reply sdk/lib/rossym/CMakeLists.txt]: Also removed a couple double quotes.

**Pull request merged**